### PR TITLE
Write onboarding docs for contributors

### PR DESCRIPTION
**Pull request by Hevia**
A nice beautiful page of docs for new contributors. Tested by Nebby & I

### CODE REVIEW

File: ONBOARDING.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+1. Create a new blank unity 3D project using the 2019.4.26.f1 version
+
+2. Install the latest versions of the following github packages, using the UnityPackageManager:
+
+- [ThunderKit](https://github.com/PassivePicasso/ThunderKit) version 8.0.3 or Greater
+- [RoR2ImportExtensions](https://github.com/risk-of-thunder/RoR2ImportExtensions) version 1.3.7 or Greater
+- [RoR2EditorKit](https://github.com/risk-of-thunder/RoR2EditorKit) version 4.0.4 or Greater
+
+
+> 2b. Sidenote, it is important that you use the versioned versions instead of the pre-release or indev versions. To select the correct version, You can click the big green “<> Code” button, and then copy the https url. You can append an “#” followed by your desired version to specify what version to download.
```
Comment (Nebby1999) [inline ONBOARDING.md:12]: Looks like the formatting here is a bit odd on the github markdown renderer, please fix

File: ONBOARDING.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Disable “Install RoR2 Editor Kit”
+
+8. After configuring your import process, click the “ThunderKit Settings” on the sidebar
+
+9. Add the exe path and hit Import (Unity might ask you to restart the project twice and this process could take awhile)
+
+10. If during the import process, the editor stops responding and doesnt continue importing, close the project, and delete any folders in the packages folder (Except for the moonstorm shared utils one) Then reopen the project and reimport
+
+>NOTE: If you ever see this image, ALWAYS hit “No Thanks”
+![](./Docs/NoThanks.png)
```
Comment (Nebby1999) [inline ONBOARDING.md:36]: The formatting here is also a bit weird, fix this as well

File: ONBOARDING.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+11. Copy `MoonstormSharedUtils/Editor/Assets/ThunderKit/Pipelines/Contributor/GenericContributor` to your `Assets` folder and configure the pipeline similar to [Starstorm 2's Build Config](https://github.com/TeamMoonstorm/Starstorm2#issues-qa)
+
+12. You should now be able to build!
+Make sure: 
+![](./Docs/makeSurePt1.png)
+That MSUManifest is selected in the build task and that the pipeline job has the correct path here so it knows to output in the plugins folder/
```
Comment (Nebby1999) [inline ONBOARDING.md:43]: Image is oddly positioned on the markdown document

File: ONBOARDING.md (hunk 1 above)
Comment (Hevia) [reply ONBOARDING.md]: Technically formatting isnt weird, its just me being weird using an uncommon markdown note style

**Pull request merged**